packages/react/src/components/proxies.ts

```diff
@@ -3,75 +3,142 @@
 /* auto-generated react proxies */
```

Comment:
Everything below this is still auto generated, correct?

Reply:
Yes

Reply:
Awesome, everything looks good to me then. Good work 💯